packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,38 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [33.0.0]
+
+### Added
+
+- Added `Mutex` lock in the `updateNftMetadata` function ([#4325](https://github.com/MetaMask/core/pull/4325))
```

Comment:
Is this an addition? Does the new mutex lock mean a new feature for the user or does it mean a change in behavior?

Reply:
IDK the actual context of this PR, but I believe it's better to be categorized as a `Changed`, after quickly reading the PR content, this does not seems to be a new feature.

I'll change this to `Changed`

Reply:
It looks like the behavior change is something like this:

```md
- Prevent unnecessary state updates when executing the `NftController`'s `updateNftMetadata` method by comparing the metadata of fetched NFTs and NFTs in state and synchronizing state updates using a mutex lock. ([#4325](https://github.com/MetaMask/core/pull/4325))
```

@sahar-fehri Would you be able to take a look at this entry and tell us whether it's accurate?

Reply:
Hello 👋 , That sounds correct, apologies for the confusion 🙏 
I wanted to introduce that mutex to synchronize the updates when the `updateNftMetadata` function is called on mobile. Having the version without the mutex was triggering many unnecessary updates.

Reply:
Thanks @sahar-fehri, @MajorLift let me use your entry then!

Reply:
Sounds good! I edited it to emphasize the key change. 

Thanks for the clarification @sahar-fehri :)

Reply:
Also this might be a Fixed rather than a Changed.

Reply:
You're right, lemme re-categorized this then :)